docker/Dockerfile-build-aggregator.jvm

```diff
@@ -0,0 +1,42 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+###
+
+# Build the project
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
+USER root
+COPY . /home/jboss
+WORKDIR /home/jboss
+RUN mvn clean package -DskipTests -Dcheckstyle.skip=true --projects :checkstyle,:notifications-parent,:notifications-aggregator
+
+# Build the container
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+ARG JAVA_PACKAGE=java-11-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.8
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install openssl curl ca-certificates ${JAVA_PACKAGE} \
+    && microdnf update \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown 1001 /deployments \
+    && chmod "g+rwX" /deployments \
+    && chown 1001:root /deployments \
+    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 540 /deployments/run-java.sh \
+    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
+
+# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+
+COPY --from=build --chown=1001 /home/jboss/backend/target/quarkus-app/ /deployments/
```

Comment:
Should be:
`COPY --from=build --chown=1001 /home/jboss/aggregator/target/quarkus-app/ /deployments/
`